src/ObserverComponent/EventMapper.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Infinityloop\ObserverComponent;
+
+class EventMapper
+{
+    private \Nette\Application\Application $application;
+    private \Nette\Caching\IStorage $storage;
+    private ?\Nette\Caching\Cache $eventMap = null;
+    private bool $debugMode;
+
+    public function __construct(
+        \Nette\Application\Application $application,
+        \Nette\Caching\IStorage $storage
+    )
+    {
+        $this->application = $application;
+        $this->storage = $storage;
+        $this->debugMode = \App\Bootstrap::isDebugMode();
```

Comment:
Remove reference to App\Bootstrap. Add setter or constrctor argument to set $debugMode.